src/main/resources/lang/default.lang

```diff
@@ -1206,6 +1206,15 @@ entities:
 	dark oak chest boat:
 		name: dark oak chest boat¦s
 		pattern: dark oak chest boat(|1¦s)
+	cherry chest boat:
+		name: cherry chest boat¦s
+		pattern: cherry chest boat(|1¦s)
+	mangrove chest boat:
+		name: mangrove chest boat¦s
+		pattern: mangrove chest boat(|1¦s)
+	bamboo chest raft:
+		name: bamboo chest raft¦s
+		pattern: bamboo chest (raft|boat)(|1¦s)
```

Comment:
```suggestion
		pattern: cherry chest boat[1:s]
	mangrove chest boat:
		name: mangrove chest boat¦s
		pattern: mangrove chest boat[1:s]
	bamboo chest raft:
		name: bamboo chest raft¦s
		pattern: bamboo chest (raft|boat)[1:s]
```

Reply:
Do you want me to apply this to all the other (chest)boat patterns as well?

Reply:
sure if you want